cors/redirect-preflight.htm

```diff
@@ -12,54 +13,62 @@
 <div id=log></div>
 <script>
 
-var req_c = 0 // Request count for cache busting and easy identifying of request in traffic analyzer
+// Request count for cache busting and easy identifying of request in traffic
+// analyzer.
+var req_c = 0;
+
+var CROSSDOMAIN_URL = CROSSDOMAIN + 'resources/cors-makeheader.py?';
 
 /*
- * Redirection with preflights
+ * Redirection with preflights.
  */
-
 function redir_preflight(code) {
     test(function() {
-        var client = new XMLHttpRequest(),
-            redirect = CROSSDOMAIN + 'resources/cors-makeheader.py?headers=x-test&' + req_c++
+        var client = new XMLHttpRequest();
+        var redirect =
+          encodeURIComponent(CROSSDOMAIN_URL + 'headers=x-test&' + req_c++);
 
-        client.open('GET', CROSSDOMAIN + 'resources/cors-makeheader.py?'
-                           + 'headers=x-test&location=' + encodeURIComponent(redirect)
-                           + '&code=' + code + '&preflight=' + code + '&' + req_c++,
-                    false)
-        client.setRequestHeader('x-test', 'test')
+        client.open('GET',
+                    CROSSDOMAIN_URL + 'headers=x-test&location=' + redirect
+                                    + '&code=' + code + '&preflight=' + code
+                                    + '&' + req_c++,
+                    false);
+        client.setRequestHeader('x-test', 'test');
         assert_throws(null, function() { client.send(null) });
 
-    },
-    'Redirect ' + code + ' on preflight')
+    }, 'Redirect ' + code + ' on preflight');
```

Comment:
So these tests are not actually checking redirects as much as they are checking that a non-2xx status code on preflights causes failure?